tests/test_decorators.py

```diff
@@ -48,6 +48,20 @@ def sample(self, bqm):
 class TestBQMIndexLabels(unittest.TestCase):
     pass
 
+class EmptyStructuredSampler(dimod.Structured):
```

Comment:
It doesn't really matter but I would either put these above `TestBQMIndexLabels` or down just above `TestBQMStructured`. I have a very weak preference for the latter.

Reply:
I had meant to put them above ``TestBQMIndexLabels``. I moved them to above ``TestBQMStructured``, why not make you happy?